lib/curator.rb

```diff
@@ -0,0 +1,70 @@
+class Curator
+  attr_reader :photographs, :artists
+
+  def initialize
+    @photographs = []
+    @artists = []
+  end
+
+  def add_photograph(photograph)
+    @photographs << photograph
+  end
+
+  def add_artist(artist)
+    @artists << artist
+  end
+
+  def find_artist_by_id(id)
+    @artists.find do |artist|
+      artist.id == id
+    end
+  end
+
+  def find_photograph_by_id(id)
+    @photographs.find do |photograph|
+      photograph.id == id
+    end
+  end
+
+  def find_photographs_by_artist(artist)
+    @photographs.find_all do |photograph|
+      photograph.artist_id == artist.id
+    end
+  end
+
+  def artists_with_multiple_photographs
+    # find all artists whose id matches the artist_id
+    # of more than one photograph
+    # call on artist id and return an array of those artist objects
+    # use find_photographs_by_artist method?
+
+    photograph_artist_ids = @photographs.map do |photograph|
+      photograph.artist_id
+    end
+
+    artist_ids = @artists.map do |artist|
+      artist.id
+    end
```

Comment:
inside this method, we have access to @photographs and we have access to @artists. We could iterate over each artist with an enumerable that will collect the artist object if we were to call a previous method we've already written above to get that artist's photos, and see if that length is greater than 1